Sources/StreamVideo/WebRTC/v2/SFU/SFUEventAdapter.swift

```diff
@@ -124,10 +150,12 @@ final class SFUEventAdapter {
                 audioLevel: level.level
             )
         }
-        //        stateAdapter.participantsUpdateSubject.send(participants)
         await stateAdapter.didUpdateParticipants(participants)
     }
 
+    /// Handles a ChangePublishQuality event.
```

Comment:
We don't have to document the private methods - unless it contains some specific info.

Reply:
We definitely don't have to but i'm using those comments to note in some cases what the function does. The rest are just for consistency.